src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Upcoming Release
 * Fixed Disable SQL AG AutoProtection.
+* Added Archive for V1 vaults.
+* Added ProtectedItemsCount in Get-AzRecoveryServicesBackupProtectionPolicy.
```

Comment:
Please move the new added change log under the top of upcoming release section.

Reply:
Done